pastel/reg_ticket.go

```diff
@@ -106,7 +106,7 @@ type AppTicket struct {
 	AlternateNSFWScores     AlternateNSFWScores `json:"alternate_nsfw_scores"`
 	ImageHashes             ImageHashes         `json:"image_hashes"`
 
-	RQIDs []string `json:"rq_ids"`
+	RQIDs []string `json:"rqids"`
```

Comment:
Please refer to the document: https://pastel.wiki/en/Architecture/Components/TicketStructures
Why did you change here?

Reply:
@danileng20 good catch. Thanks!